aleph/views/roles_api.py

```diff
@@ -51,12 +54,26 @@ def suggest():
       - Role
     """
     require(request.authz.logged_in)
-    return jsonify(
-        {
-            "results": [],
-            "total": 0,
-        }
+    parser = QueryParser(request.args, request.authz, limit=10)
+    if parser.prefix is None or len(parser.prefix) < 3:
```

Comment:
I guess we could raise minimum length to 5 or 6 here, but it's a minor thing.

Reply:
Done!